cmds/core/stat/run_darwin.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2025 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"syscall"
+	"time"
+)
+
+func fromTimespec(ts syscall.Timespec) string {
+	t := time.Unix(ts.Sec, ts.Nsec)
+	return t.Format("Jan  2 15:04:05 2006")
+}
+
+// device inode perm nlink uid gid rdev size atime mtime
+// ctime btime blksize blocks st_flags filename
+const defFormat = "%d %d %s %d %d %d %d %d %q %q %q %q %d %d %d %s\n"
+
+func run(stdout io.Writer, stderr io.Writer, files ...string) int {
+	var errCode int
```

Comment:
In your current implementation, passing no files into the variadic function will let it return without an error. The coreutils implementation returns `1` if run without any files; this behaviour should be the same to guarantee scripting behavior etc.

Reply:
I think it because you are using linux:
```
stat$ stat
stat: missing operand
Try 'stat --help' for more information.
```

but on unix it works differently it stats stdin and this is not implemented yet.